src/pal/src/exception/remote-unwind.cpp

```diff
@@ -48,7 +48,9 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include "pal.h"
 #include <dlfcn.h>
 
+#ifndef __linux__
```

Comment:
Why this ifdef? I would think this should be done everywhere.

Reply:
This should be `#ifndef __APPLE__` now. The OSX libunwind doesn't support remote unwind.